c7n/resources/ses.py

```diff
@@ -34,3 +35,15 @@ class resource_type(TypeInfo):
         'describe': DescribeConfigurationSet
     }
 
+
+@resources.register('ses-email-identity')
```

Comment:
This is gonna capture email identities. From a control standpoint I think the goal is to make sure `identity.DkimAttributes.SigningEnabled` is True. @kapilt, i'm not sure if this also falls into the category of capturing personal data. Would love to have your feedback here. Thanks

Reply:
The captured email identities will only be those that are verified and used to send emails, not receive them. IMO I think that is acceptable since the emails captured should be corporate accounts and not personal.

https://docs.aws.amazon.com/ses/latest/dg/verify-addresses-and-domains.html